contents/blog/the-posthog-array-1-40-0.md

```diff
@@ -0,0 +1,71 @@
+---
+date: 2022-08-29
+title: 'Array 1.40.0: What's new in PostHog?'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Release notes', 'Product updates']
+featuredImage: ../images/blog/posthog-array-blog.png
+featuredImageType: full
+---
+
+Want to know more about what we're up to? [Subscribe to HogMail, our newsletter](/newsletter), which we send out every two weeks!
+
+> Running a self-hosted instance? Check out our [guide to upgrading PostHog](/docs/runbook/upgrading-posthog).
+
+## PostHog 1.40.0 release notes
+
+**Release highlights:**
+
+- [Highlight One](#highlight-one)
+- [Highlight Two](#highlight-two)
+- [Highlight Three](#highlight-three)
+
+### Highlight One
+
+Some copy goes here. 
```

Comment:
@neilkakkar @EDsCODE It would be great if, as you get an idea what will be included, you can add just a single sentence here for each major feature. @andyvan-ph and I will take it from there!